examples/00-mapdl-examples/composite_dcb.py

```diff
@@ -358,7 +359,7 @@
 add_op_cohesive = dpf.operators.math.add(fieldA=mesh_field_cohesive)
 
 # Instantiate a PyVista plotter and start the creation of a GIF
-plotter = pv.Plotter(notebook=False, off_screen=True)
+plotter = pv.Plotter(window_size=[640, 480])
```

Comment:
Removing ``notebook=False, off_screen=True``?? why?

Reply:
This was also part of the testing, reverted.